mllib/src/main/scala/org/apache/spark/ml/param/params.scala

```diff
@@ -249,6 +249,16 @@ object ParamValidators {
   def arrayLengthGt[T](lowerBound: Double): Array[T] => Boolean = { (value: Array[T]) =>
     value.length > lowerBound
   }
+
+  /** Check if more than one param in a set of exclusive params are set. */
+  def checkExclusiveParams(model: Params, params: String*): Unit = {
+    if (params.filter(paramName => model.hasParam(paramName) &&
```

Comment:
Why is this checking to see if the Param belongs to the Model?  If this method is called with irrelevant Params, shouldn't it throw an error?

Reply:
The purpose of this method is to check if more than one Params are set among some exclusive Params within a Model. Is it useful to put an irrelevant Param into the exclusive Params to check? As we already know what Params the model has, it sounds like we want to check an irrelevant Param that we already know non-existing?